libs/langchain-azure-openai/src/embeddings.ts

```diff
@@ -0,0 +1,140 @@
+import { Embeddings } from "@langchain/core/embeddings";
+import {
+  type OpenAIClientOptions as AzureOpenAIClientOptions,
+  OpenAIClient as AzureOpenAIClient,
+  AzureKeyCredential,
+} from "@azure/openai";
+import { getEnvironmentVariable } from "@langchain/core/utils/env";
+import {
+  KeyCredential,
+  TokenCredential,
+  isTokenCredential,
+} from "@azure/core-auth";
+import { chunkArray } from "@langchain/core/utils/chunk_array";
+import { AzureOpenAIInput, AzureOpenAIEmbeddingsParams } from "./types.js";
+
+export class AzureOpenAIEmbeddings
+  extends Embeddings
+  implements AzureOpenAIEmbeddingsParams, AzureOpenAIInput
+{
+  modelName = "text-embedding-ada-002";
```

Comment:
Should we add a `dimensions` property here too for v3?

Reply:
We could add the `dimensions` property as an update later after the initial commit. I would like to do the required testing around it and ensure that no breaks in the code. Also, I will be contributing on a consistent basis to this repository. So, we could add this later